client/my-sites/checkout/checkout-thank-you/index.tsx

```diff
@@ -573,6 +567,40 @@ export class CheckoutThankYou extends Component<
 			/* eslint-enable wpcalypso/jsx-classname-namespace */
 		}
 
+		/** REFACTORED REDESIGN **/
+
+		if ( isRefactored( this.props ) ) {
```

Comment:
Introduced the function `isRefactored()` so that we can gradually port over the redesigned pages (`isRedesignV2()`) to the new framework of rendering page using `<ThankYouV2>`.